lxd/storage/backend_lxd.go

```diff
@@ -5214,6 +5221,11 @@ func (b *lxdBackend) CreateCustomVolumeSnapshot(projectName, volName string, new
 		return err
 	}
 
+	// ISO images are read-only, therefore snapshots aren't necessary.
```

Comment:
Can we make this logic more defensive in general, so define the content types that are supported (possibly using a switch statement) and then if its anything return an error saying the specific content type is not supported for the action.